pkg/controller/persistentvolume/types.go

```diff
@@ -26,10 +26,25 @@ import (
 )
 
 const (
+	// A PVClaim can request a quality of service tier by adding this annotation.  The value of the annotation
+	// is arbitrary.  The values are pre-defined by a cluster admin and known to users when requesting a QoS.
+	// For example tiers might be gold, silver, and tin and the admin configures what that means for each volume plugin that can provision a volume.
+	qosProvisioningKey = "volume.experimental.kubernetes.io/quality-of-service"
+	// When a PVClaim requests a QoS tier, the "provisionable" annotation is applied so that the provisioner will process the request for a new volume.
+	provisionableKey = "volume.experimental.kubernetes.io/provisionable"
+	// PVClaims for which volumes have been provisioned are given this annotation so that the provisioner ignores them on subsequent sync periods.
+	provisionedKey = "volume.experimental.kubernetes.io/provisioned"
 	// A PV created specifically for one claim must contain this annotation in order to bind to the claim.
 	// The value must be the namespace and name of the claim being bound to (i.e, claim.Namespace/claim.Name)
 	// This is an experimental feature and likely to change in the future.
-	createdForKey = "volume.extensions.kubernetes.io/provisioned-for"
+	provisionedForKey = "volume.experimental.kubernetes.io/provisioned-for"
+
+	// Name of a tag attached to a real volume in cloud (e.g. AWS EBS or GCE PD)
+	// with namespace of a persistent volume claim used to create this volume.
+	cloudVolumeCreatedForNamespaceTag = "kubernetes.io/created-for/PVC/Namespace"
+	// Name of a tag attached to a real volume in cloud (e.g. AWS EBS or GCE PD)
+	// with name of a persistent volume claim used to create this volume.
+	cloudVolumeCreatedForNameTag = "kubernetes.io/created-for/PVC/Name"
```

Comment:
swithed to lowercase, `kubernetes.io/created-for/pvc/namespace` and `kubernetes.io/created-for/pvc/name`